sources/Outputs/src/Par.cpp

```diff
@@ -86,6 +86,8 @@ Par::updateSignalNGenerator(boost::shared_ptr<parameters::ParametersSet> set) {
   set->addReference(helper::buildReference("generator_Q0Pu", "q_pu", "DOUBLE"));
   set->addReference(helper::buildReference("generator_U0Pu", "v_pu", "DOUBLE"));
   set->addReference(helper::buildReference("generator_UPhase0", "angle_pu", "DOUBLE"));
+  set->addReference(helper::buildReference("generator_Pref0Pu", "targetP_pu", "DOUBLE"));
+  set->addReference(helper::buildReference("generator_Uref0Pu", "targetV_pu", "DOUBLE"));
```

Comment:
Pref0Pu and Uref0Pu should be added also in writeGenerator function

Reply:
Same remark as Marco

Reply:
This has been corrected in a recent commit

Reply:
I saw it just after my comment, my bad!

Reply:
No problem